Makefile

```diff
@@ -10,6 +10,8 @@ DOCKERS_DEV = $(addprefix docker_dev_,$(SERVICES))
 CGO_ENABLED ?= 0
 GOOS ?= linux
 
+DOCKER_IMG_NAME ?= metav
```

Comment:
Please clean up this commit, I think we don't need your tests in our commit history ;)

Reply:
fixed sorry